doc/version.json

```diff
@@ -1,11 +1,13 @@
 [
     {
-        "version": "dev",
+        "name": "1.4.dev0 (dev)",
+        "version": "1.4.dev0",
         "url": "https://skrub-data.org/dev/"
     },
     {
-        "version": "stable",
+        "name": "1.3.0 (stable)",
+        "version": "1.3.0",
```

Comment:
I think this should fix the fact that the banner appears even on the stable branch docs, see https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/announcements.html#version-warning-banners